Source/Initialization/PlasmaInjector.cpp

```diff
@@ -292,6 +306,10 @@ PlasmaInjector::PlasmaInjector (int ispecies, const std::string& name)
 #if WARPX_DIM_XZ
         num_particles_per_cell_each_dim.push_back(1);
 #endif
+#if WARPX_DIM_1D_Z
+        num_particles_per_cell_each_dim.push_back(1);
```

Comment:
I am confused about this, why twice the push_back? If correct, maybe add a comment inline? :)

Reply:
In 1D, only one number is required for num_particles_per_cell_each_dim. If three are provided, 2nd and third is overwritten by 1. I added a comment inline. :)